src/_includes/feature_lists/cloud.njk

```diff
@@ -19,6 +19,9 @@
         }, {
             label: "Role-Based Access Control",
             values: ['check', 'check', 'check']
+        }, {
+            label: "Edge Device Management",
```

Comment:
It would be great if this could be a link to the docs, once those are written.

Reply:
@ZJvandeWeg add this is a new issue/feature request please

Reply:
https://github.com/flowforge/flowforge/issues/653